cookbook/form/data_transformers.rst

```diff
@@ -17,6 +17,11 @@ You could try to do this in your controller, but it's not the best solution.
 It would be better if this issue were automatically converted to an Issue object.
 This is where Data Transformers come into play.
 
+.. caution::
+
+    When a field has the :doc:`inherit_data </reference/forms/types/options/inherit_data>`
```

Comment:
you can't link to included files. So either link to an inherit_data option of the Form element reference (create a new ref pointer) or remove interlinking here.